MApSS-DEV/Maui.PDFView
# Sam mapssiv 1316 prototype support in standalone

![image](https://github.com/MApSS-DEV/Maui.PDFView/assets/40006493/45469e1e-0cc7-49f8-91bf-e120880f9975)
![image](https://github.com/MApSS-DEV/Maui.PDFView/assets/40006493/79c2be6a-ebe5-4363-8168-bbfe7d85ac8d)

URL to Navy samples:
https://github.com/MApSS-DEV/mynavy_uniforms-content/tree/main/PDFs

To test app:

> Rename you test pdf for either Pdf1 or Pdf2. (See Example/Resources/PDF
> Validate or change Example.proj as follows:
	<ItemGroup>
		<None Remove="Resources\PDF\pdf1.pdf" />
		<None Remove="Resources\PDF\pdf2.pdf" />
	</ItemGroup>

	<ItemGroup>
		<EmbeddedResource Include="Resources\PDF\pdf1.pdf" />
		<EmbeddedResource Include="Resources\PDF\pdf2.pdf" />
	</ItemGroup>

> Pdf __pdf2.pdf will crash the system. This happens in the XAML, so not error is thrown. The add just goes away.
